src/pylexibank/providers/sndcmp.py

```diff
@@ -0,0 +1,359 @@
+import pathlib
+import zipfile
+import attr
+import json
+import re
+
+from collections import OrderedDict
```

Comment:
This is also a stdlib import, thus should be in the first block of imports. We also try to always import the base module/package from stdlib, i.e.
```python
import collections
```